packages/core/admin/ee/server/services/user.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const { pipe, castArray, map, toNumber } = require('lodash/fp');
+
+/** Checks if ee disabled users list needs to be updated
+ * @param {string} id
+ * @param {object} input
+ */
+const shouldUpdateEEDisabledUsersList = async (id, input) => {
```

Comment:
I think it's clear if you pass the user directly, where you already have the id.
```suggestion
const shouldUpdateEEDisabledUsersList = async (updatedUser) => {
```